libraries/aws_db_subnet_groups.rb

```diff
@@ -0,0 +1,57 @@
+# frozen_string_literal: true
+
+require 'aws_backend'
+
+class AwsDbSubnetGroups < AwsResourceBase
+  name 'aws_db_subnet_groups'
+  desc 'Verifies settings for a Subnet Groups in bulk'
+  example '
+    describe aws_db_subnet_groups do
+      it { should exist }
+    end
+  '
+
+  attr_reader :table, :subnet_rows
+  FilterTable.create
+             .register_column(:db_subnet_group_names,                field: :db_subnet_group_name)
+             .register_column(:db_subnet_group_descriptions,         field: :db_subnet_group_description)
+             .register_column(:vpc_ids,                              field: :vpc_id)
+             .register_column(:subnet_group_status,                  field: :subnet_group_status)
+             .register_column(:subnets,                              field: :subnets)
+             .register_column(:db_subnet_group_arns,                 field: :db_subnet_group_arn)
+             .install_filter_methods_on_resource(self, :table)
+
+  def initialize(opts = {})
+    super(opts)
+    validate_parameters
+    @table = fetch_data
+  end
+
+  def fetch_data
+    subnet_group_rows = []
+    pagination_options = {}
+    catch_aws_errors do
+      resp = @aws.rds_client.describe_db_subnet_groups(pagination_options)
+      return [] if !resp || resp.empty?
+      resp.db_subnet_groups.each do |subnet_group|
+        subnet_group_rows += [{ db_subnet_group_name:           subnet_group.db_subnet_group_name,
+                                db_subnet_group_description:    subnet_group.db_subnet_group_description,
+                                vpc_id:                         subnet_group.vpc_id,
+                                subnet_group_status:            subnet_group.subnet_group_status,
+                                subnets:                        return_subnets(subnets: subnet_group.subnets),
+                                db_subnet_group_arn:            subnet_group.db_subnet_group_arn }]
+      end
+    end
+    @table = subnet_group_rows
+  end
+
+  def return_subnets(subnets:)
```

Comment:
This is a great idea and it should be in the singular resource as well.
This is what I meant in my previous comment here: https://github.com/inspec/inspec-aws/pull/221/commits/37c54a6e3101709f1ba256439534dc1a3643f1ee?file-filters%5B%5D=.rb#r428678279